db/R/dbfiles.R

```diff
@@ -34,16 +34,29 @@
 dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate, mimetype, formatname, parentid=NA, con, hostname=fqdn()) {
   
   name <- basename(in.path)
-  filename <- file.path(in.path,in.prefix)
+  filename <- file.path(in.path, in.prefix)
   
   if (hostname == "localhost") hostname <- fqdn();
   
+  mimetypeid <- get.id("mimetypes", "type_string", mimetype, con) #paste0("SELECT id FROM mimetypes where type_string = '", mimetype, "';")
```

Comment:
I like the get.id function, would it make sense to also have it create the entry if it does not exist?